src/test-error/plain-text-decorator.js

```diff
@@ -0,0 +1,27 @@
+const ACTUAL_MSG_PREFIX = 'Actual:   ';
+
+function escapeNewLines (str) {
+    return str.replace(/(\r\n|\n|\r)/gm, '\\n');
+}
+
+export default {
+    'actual': (str) => ACTUAL_MSG_PREFIX + escapeNewLines(str),
+
+    //NOTE: add gaps with length equal the length of actual prefix
+    //''    =>  'Actual:   '
+    //'^'   =>  '         ^'
+    'diff-marker': (str) => str ? ACTUAL_MSG_PREFIX.replace(/./g, ' ') + str : '',
+
+    'error-category': () => '',
+
+    'expected': (str) => 'Expected: ' + escapeNewLines(str),
```

Comment:
I thought prefixes like `Expected` will be built-in in template and you will not need to insert them in decorator. You just need to decorate `Expected` word.